.github/dependabot.yml

```diff
@@ -12,14 +12,25 @@ updates:
     labels:
       - "approved"
       - "lgtm"
-      - "dependencies"
+      - "dependabot"
       - "go"
     commit-message:
       prefix: "NO-ISSUE"
+    groups:
+      go-dependencies:
+        patterns:
+          - "*"
 
   - package-ecosystem: "docker"
     directory: "/"
     schedule:
       interval: "weekly"
+    labels:
+      - "dependabot"
+      - "docker"
```

Comment:
@osherdp any reasons we don't lgtm+approve docker updates?

Reply:
yeah those updates are basically irrelevant I'm afraid
because when the base-image changes we usually need to do changes in ``openshift/release`` repo and in the D/S Dockerfiles, and it's rarely trivial changes

Reply:
Maybe we even prefer removing those bumps as they only act as reminders. wdyt @adriengentil ?

Reply:
well, Ithink it's fine, as you say it, it acts as a reminder :)